pubspec.yaml

```diff
@@ -6,11 +6,9 @@ environment:
 dev_dependencies:
   build_runner: ^1.1.2
   code_excerpt_updater:
-    git: https://github.com/chalin/code_excerpt_updater.git
-    # path: ../code_excerpt_updater
+    git: https://github.com/RedBrogdon/code_excerpt_updater.git
```

Comment:
Thank you for this, though can't this tool live in a flutter-owned repo?

Reply:
Yes. We just nee to pick one, really. Would the site-shared repo be a good place?

CC: @kwalrath

Reply:
site-shared sgtm